Modules/IO/MINC/src/itkMINCImageIO.cxx

```diff
@@ -460,6 +460,14 @@ MINCImageIO::ReadImageInformation()
   Matrix<double, 3, 3> dir_cos{};
   dir_cos.SetIdentity();
 
+  // Conversion matrix for MINC PositiveCoordinateOrientation RAS (RightToLeft,AnteriorToPosterior,SuperiorToInferior)
+  // to ITK PositiveCoordinateOrientation LPS (LeftToRight,PosteriorToInferior,SuperiortoInferior)
+  Matrix<double, 3, 3> RAS_tofrom_LPS;
```

Comment:
ITK style stipulates that variables should not use underscores; I am well aware that four lines above we have one example, but given that this is new code, let's stick to conventions so that the code becomes increasingly consistent at this.